Dockerfile

```diff
@@ -48,8 +49,15 @@ RUN docker-php-ext-install   pcntl \
         bcmath \
         calendar \
         pdo_mysql \
+        sockets \
```

Comment:
have you tried the docker compose by specifying the image instead of building it ?